test/unit/loan/LendingTerm.t.sol

```diff
@@ -31,7 +31,7 @@ contract LendingTermUnitTest is ECGTest {
     // LendingTerm params
     uint256 constant _CREDIT_PER_COLLATERAL_TOKEN = 2000e18; // 2000, same decimals
     uint256 constant _INTEREST_RATE = 0.10e18; // 10% APR
-    uint256 constant _MAX_DELAY_BETWEEN_PARTIAL_REPAY = 63115200; // 2 years
+    uint256 constant _MAX_DELAY_BETWEEN_PARTIAL_REPAY = 31557600; // 1 years
```

Comment:
Here there was a problem: before the parameters checks were done in the LendingTermFactory so this unit test was able to create a lending term with a 2 year min delay between partial repay

Now that the parameter tests are in the lending term contract, the max delay between partial repay is enforced to 1 year